### DISCUSSION TITLE
YOLO badge shows private contribution information

### DISCUSSION THREAD

**Discussion by chardan** [community/community#19844] [Other Feature Feedback, Questions, & Ideas]
The YOLO badge reveals information about private contributions. I hope that can be fixed!

**Comment by sinkaroid** [comment]
It's just revealed to the owner and users that have access itself, In public it will changed to "inaccessible"  
This is what it looks like when I visit your profile
![Screenshot_587](https://user-images.githubusercontent.com/12372481/176971545-5f73fe93-2804-440c-a6eb-ee470b998d80.png)

**Reply by chardan** [reply]
Thank you for clarifying that!
